docs/source/markdown/options/uidmap.container.md

```diff
@@ -76,6 +76,24 @@ Every additional range is added sequentially afterward:
 | 1                     | $FIRST_RANGE_ID        | $FIRST_RANGE_LENGTH |
 | 1+$FIRST_RANGE_LENGTH | $SECOND_RANGE_ID       | $SECOND_RANGE_LENGTH|
 
+By default providing **--uidmap** or **--gidmap** replaces the whole mapping.
+If only one of those two are given, by default the other one is copied,
+so if you want to change only one of the two, you should provide both.
+
+When running as rootless, you may want to map a specific host group
+that additionally has ben subordinated to you at _/etc/subgid_, without
+specifying the rest of the mapping. This can be done by passing
+**--gidmap=+*container_gid*:*@host_gid*:1**. This uses the *+* sign
+to specify that we want to extend the default mapping, and not replace
+it. It also uses the *@* sign to specify that the mapping refers to the
+host namespace, which is the parent of the intermediate namespace. For instance,
+if you belong to the additional group 1002 and that group is subordinated
+to you with `usermod --add-subgids 1002-1002 $USER`, you can map that
+group into the container with: **--gidmap=+10000:@1002:1**. If you
+combine this mapping, with **--group-add=keep-groups**, your user in the
+container will belong to group 10000 and files that belong to the group
+1002 in the host will appear as owned by 10000 inside the container.
+
```

Comment:
Please no pro-nouns in the man pages.  I'll try to work out the "you" and "your" words that I'm seeing quickly within a comment or three.